supplier_app/models.py

```diff
@@ -339,8 +340,54 @@ def get_taxpayer_identifier(self):
         return self.cuit
 
 
+class TaxPayerUnitedStates(TaxPayer):
+    taxpayer_id_number = models.CharField(
+        max_length=9,
+        unique=True,
+        validators=[
+            RegexValidator(
+                r'^[0-9]+$',
+                message=_('Id number must only have numbers'),
+                code='invalid_id'
+            ),
+            MaxLengthValidator(9),
+            MinLengthValidator(9),
+        ]
+    )
```

Comment:
both EIN and SSN have same validation rules?

Reply:
Yes